client/src/partials/subsidies/subsidies.html

```diff
@@ -118,40 +118,41 @@
-                <label class="control-label" for="description">{{ 'FORM.LABELS.DESCRIPTION' | translate }}</label>
+                <label class="control-label" for="description" translate> FORM.LABELS.DESCRIPTION </label>
                 <textarea class="form-control" id="description" name="description" ng-maxlength="SubsidyCtrl.maxLength" ng-model="SubsidyCtrl.subsidy.description" rows="3"></textarea>
                 <div class="help-block" ng-messages="CreateForm.description.$error" ng-show="CreateForm.$submitted">
                   <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
                 </div>
               </div>
 
               <div class="form-group">
-                <button class="btn btn-default" id="submit-subsidy" type="submit" data-method="submit">
-                  {{ "FORM.BUTTONS.SUBMIT" | translate }}
-                </button>
-                <button class="btn btn-default" type="button" ng-click="SubsidyCtrl.cancel()" data-method="cancel">
-                  {{ "FORM.BUTTONS.CANCEL" | translate }}
+                <bh-loading-button loading-state="CreateForm.$loading">
+                  <span translate>FORM.BUTTONS.SUBMIT</span>
+                </bh-loading-button>
+
+                <button class="btn btn-default" type="button" ng-click="SubsidyCtrl.cancel()" data-method="cancel" translate>
+                  FORM.BUTTONS.CANCEL
                 </button>
               </div>
             </form>
           </div>
         </div>
 
         <div class="panel panel-primary" ng-switch-when="update">
-          <div class="panel-heading">{{ 'SUBSIDY.UPDATING_SUBSIDY' | translate }}</div>
+          <div class="panel-heading" translate> SUBSIDY.UPDATING_SUBSIDY </div>
           <div class="panel-body">
             <form name="UpdateForm" ng-submit="SubsidyCtrl.submit(UpdateForm.$invalid)" bh-form-defaults novalidate>
 
               <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.label.$invalid }">
-                <label class="control-label">{{ 'FORM.LABELS.NAME' | translate }}</label>
+                <label class="control-label" translate> FORM.LABELS.NAME </label>
                 <input type="text" class="form-control" name="label" autocomplete="off" ng-maxlength="SubsidyCtrl.length250" ng-model="SubsidyCtrl.subsidy.label" required>
                 <div class="help-block" ng-messages="UpdateForm.label.$error" ng-show="UpdateForm.$submitted">
                   <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
                 </div>
               </div>
 
               <div class="form-group" ng-class="{ 'has-error' : UpdateForm.$submitted && UpdateForm.value.$invalid }">
-                <label class="control-label">{{ 'FORM.LABELS.VALUE' | translate }}</label>
+                <label class="control-label" translate> FORM.LABELS.VALUE </label> (%)
```

Comment:
This coloring is a bit weird since the `(%)` is not in bold, but the other characters are.  Why don't you change this too:
```html
<label class="control-label">
  <span translate>FORM.LABELS.VALUE</span> (%)
</label>
```

That way, the whole label shares the `control-label` styles.